.gitignore

```diff
@@ -6,6 +6,7 @@
 **/.settings/*
 **/.prefs
 **/.classpath
+**/.factorypath
```

Comment:
Does [this](https://github.com/strimzi/strimzi-kafka-operator/blob/ecc2a64189c04658d629d32beb4ffbc29fa35ba3/.gitignore#L20) cover it already? It was added in the mean time by Matthias.

Reply:
Yes, I didn't sync with this morning merges :-)